src/main.ts

```diff
@@ -28,12 +28,13 @@ import { ClusterClient } from './gkeClient';
 async function run(): Promise<void> {
   try {
     // Get inputs
-    const name = getInput('cluster_name', { required: true });
+    const clusterName = getInput('cluster_name', { required: true });
     const location = getInput('location', { required: true });
     const credentials = getInput('credentials');
     const projectId = getInput('project_id');
     const useAuthProvider = getBooleanInput('use_auth_provider');
     const useInternalIP = getBooleanInput('use_internal_ip');
+    const contextName = getInput('context_name') || `gke_${projectId}_${location}_${clusterName}`;
```

Comment:
🐉 for discussion. This changes the default to match gcloud's default naming, but it changes the existing behavior for default naming.

Reply:
IMO switching to gcloud default should be okay. However `projectId` is not a required input and maybe empty here as we support discovery via `GCLOUD_PROJECT`

Reply:
Good point. Can we merge https://github.com/google-github-actions/get-gke-credentials/pull/160 first since it has the nicer project ID extraction at this layer?

Reply:
@sethvargo Just merged #160

Reply:
Done - this logic gets a little circular, so please take a close look